training-diary.code-workspace

```diff
@@ -0,0 +1,47 @@
+{
+  "folders": [
+    {
+      "name": "🌱 root",
+      "path": "."
+    },
+    {
+      "name": "📦 Backend",
+      "path": "./backend"
+    },
+    {
+      "name": "📦 Frontend",
+      "path": "./frontend"
+    }
+  ],
+  "settings": {
+    "typescript.tsdk": "./frontend/node_modules/typescript/lib",
+    "emmet.includeLanguages": {
+      "postcss": "css"
+    }
+  },
+  "extensions": {
+    "recommendations": [
+      "editorconfig.editorconfig",
+      "dbaeumer.vscode-eslint",
+      "eamodio.gitlens",
+      "bradlc.vscode-tailwindcss",
+      "cpylua.language-postcss",
+      "xdebug.php-debug"
+    ]
+  },
```

Comment:
When we open the workspace in the VSCode there will be a small popup with a suggestion to install recommended extensions. I highly recommend to do that.